package.json

```diff
@@ -23,5 +23,8 @@
     "devDependencies": {
         "dtslint": "github:Microsoft/dtslint#production",
         "types-publisher": "Microsoft/types-publisher#production"
+    },
+    "dependencies": {
+        "set-cookie-parser": "^2.2.1"
```

Comment:
Why?

Reply:
I am so sorry this is a mistake, I added this to run tests. I'll remove it.